### ISSUE TITLE
Create separate DMCA takedown form (with counter-notification form)

### ISSUE THREAD

**Issue by sarahpearson** [creativecommons/list#336]
Here is an example: https://www.kickstarter.com/help/forms/dmca/new
Here is CC's DMCA agent info: http://creativecommons.org/dmca

What is the best way to do this? Should I write something up, or is the example enough for you to go on for now?

**Comment by perpetuatingcuriosity** [comment]
Cool! I like the Kickstarter example. Is there currently a similar web form available on the CC site? We could just link to it from the Android app.

**Comment by perpetuatingcuriosity** [comment]
@sarahpearson If you could write up a separate issue up a bit about how you see the process of flagging a photo working, that would be awesome. 

E.g.: “I don’t want to see this” but hides it from the feed. If multiple people hide it, it gets flagged, etc.

I think it would be great to open it up for discussion and help us figure out how best to implement flagging on the back end + in the app.

**Comment by sarahpearson** [comment]
No, we don't have a web form yet for the CC site, but we have talked about it. Would the simplest route be for us to create that and then have you link to it from the app then?

**Comment by perpetuatingcuriosity** [comment]
Yeah that sounds good :+1:
